content/components/pipeline/_index.en.md

```diff
@@ -9,7 +9,37 @@ description: "Data stream processing at high rate and low memory consuming"
 
 {{< feature-state for_mw_version="0.1" state="alpha" >}}
 
-### Definition of a Pipeline
+### Definition
+A pipeline is a series of processes that filter or transform data.
+A pipeline consists of zero, one or more stages/processes. 
+The first process takes raw input data, in fact something, 
```

Comment:
`in fact something` trouver comment le dire autrement

Reply:
Modifié